static/js/publisher/release/releasesController.js

```diff
@@ -3,13 +3,17 @@ import PropTypes from 'prop-types';
 import 'whatwg-fetch';
 
 import RevisionsTable from './revisionsTable';
+import RevisionsList from './revisionsList';
 import Notification from './notification';
 
 export default class ReleasesController extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
+      // default to latest track
+      currentTrack: 'latest',
```

Comment:
we need to figure out a way to set a 'default' before it's added to the rest of the snap stack - this won't have the desired effect for node. Maybe an exceptions mapping flask-side that is exposed to all templates, so the default 'default' is 'latest' but we can override.